cdk/src/open-data-platform/root-stack.ts

```diff
@@ -1,8 +1,9 @@
-import { Stack, StackProps } from 'aws-cdk-lib';
-import { Construct } from 'constructs';
+import {Stack} from 'aws-cdk-lib';
+import {Construct} from 'constructs';
 import * as util from '../util';
-import { DataPlaneStack } from './data-plane/data-plane-stack';
-import { NetworkStack } from './network/network-stack';
```

Comment:
I think we might have a different linter configured, which is causing these style changes. Let's figure that out in standup or our 1:1.